src/styles/themes/light.css

```diff
@@ -24,6 +24,9 @@
   --black: #000000;
 
   /***** Logical Palette *****/
+  --DARK: ;
+  --LIGHT: initial;
+  --OLED: ;
```

Comment:
I'd like if you could get rid of these, they encourage a practice we're trying to avoid (component styles being theme-aware)

Reply:
These are removed as well !